src/codegeneration/ClassTransformer.js

```diff
@@ -107,15 +114,40 @@ function classCall(func, object, staticObject, superClass) {
       `($traceurRuntime.createClass)(${func}, ${object}, ${staticObject})`;
 }
 
+function methodNameFromToken(token) {
```

Comment:
Token is what the scanner produces. Trees is what the parser produces and a tree can contain other trees or tokens (for leafs).